docs/docusaurus/docs/guides/setup/configuring_metadata_stores/how_to_configure_a_metricsstore.md

```diff
@@ -3,33 +3,35 @@ title: How to configure and use a MetricStore
 ---
 import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
 
-Saving <TechnicalTag tag="metric" text="Metrics" /> during <TechnicalTag tag="validation" text="Validation" /> makes it easy to construct a new data series based on observed dataset characteristics computed by Great Expectations. That data series can serve as the source for a dashboard or overall data quality metrics, for example.
+:::note Note:
+Metric storage is an **experimental** feature.
+:::
+
+A `MetricStore` is a <TechnicalTag tag="store" text="Store" /> that stores Metrics computed during Validation. A `MetricStore` tracks the `run_id` of the Validation and the <TechnicalTag tag="expectation_suite" text="Expectation Suite" /> name in addition to the Metric name and Metric kwargs.
 
-Storing metrics is still an **experimental** feature of Great Expectations, and we expect configuration and capability to evolve rapidly.
+Saving <TechnicalTag tag="metric" text="Metrics" /> during <TechnicalTag tag="validation" text="Validation" /> lets you construct a new data series based on observed dataset characteristics computed by Great Expectations. A data series can serve as the source for a dashboard, or overall data quality metrics.
 
 ## Prerequisites
 
 - [A Great Expectations instance](/docs/guides/setup/setup_overview)
 - Completion of the [Quickstart](tutorials/quickstart/quickstart.md)
```

Comment:
I think this is an invalid link -- we may need to do an update to `_prerequisites.jsx` since I believe that's where this is being generated from and I'm seeing it in multiple docs.